features/step-definitions/draft_letter.steps.js

```diff
@@ -0,0 +1,37 @@
+const { When, Then } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const SecurePage = require("../pageobjects/secure.page");
+const securePage = new SecurePage();
+const NewLetterPage = require("../pageobjects/new_letter.page");
+const newLetterPage = new NewLetterPage();
+const DraftsPage = require("../pageobjects/drafts.page");
+const draftsPage = new DraftsPage();
+
+When(/^I wait "(.*)" seconds$/, async (sec) => {
+  await browser.pause(`${sec}000`);
+});
+
+Then(/^I expect Letter is saved automatically$/, async () => {
```

Comment:
the same as previous - all text should be as variables form feature files
do not use such strings inside
you feature file should be easy to modified
so if the message of text will be updated - it is easy to update it in feature file, not in steps
so use something like
I expect "Saved text" is displayed on New Letter page